spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc

```diff
@@ -100,11 +100,33 @@ autoBindDlq::
   Whether to automatically declare the DLQ and bind it to the binder DLX.
 +
 Default: `false`.
+bindQueue::
+  Whether to bind the queue to the destination exchange; set to false if you have set up your own infrastructure and have previously created/bound the queue.
++
+Default: `true`.
+declareExchange::
+  Whether to declare the exchange for the destination.
++
+Default: `true`.
+delayedExchange::
+  Whether to declare the exchange as a `Delayed Message Exchange` - requires the delayed message exchange plugin on the broker.
+  The `x-delayed-type` argument is set to the `exchangeType`.
++
+Default: `false`.
 durableSubscription::
   Whether subscription should be durable.
 Only effective if `group` is also set.
 +
 Default: `true`.
+exchangeRoutingKey::
```

Comment:
I'll give it some thought; we're in a bit of a "bind" here 😄 

The problem is we're configuring the binder and these properties are nothing to do with the binder, they are about binding the queue to the exhange.

That's why I used `bindQueue` above; I thought `bind` alone might be confusing.